Marlin/Configuration.h

```diff
@@ -1378,7 +1378,7 @@
  * (Automatically enables USE_PROBE_FOR_Z_HOMING.)
  */
 #if BOTH(KNUTWURST_BLTOUCH, KNUTWURST_4MAXP2)
-  #define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
+  //#define Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN
```

Comment:
Where is the bltouch connected if not to the Z_MIN?

Reply:
Wiring was taken from Ai3Mega:
https://github.com/MNieddu91/Marlin-AI3M-BLTouch/wiki/Installation-BLTouch